packages/@sfx/sayt-driver-plugin/test/unit/browser/sayt-driver-plugin.test.ts

```diff
@@ -0,0 +1,193 @@
+import { expect, spy, stub } from '../../utils';
+import { SaytDriverPlugin } from '../../../src/index';
+import { DomEventsPlugin } from '../../../../dom-events-plugin/src/index';
+import { SaytPlugin } from '../../../../sayt-plugin/src';
+
+describe('Sayt Driver Plugin', () => {
+  let Driver: any;
+
+  beforeEach(() => {
+    Driver = new SaytDriverPlugin();
+    Driver.register({
+      'dom_events': new DomEventsPlugin().register({}),
+      'sayt': new SaytPlugin({
+        subdomain: 'macystest',
+        https: true,
+      }).register(),
+    });
+  });
+
+  describe('get metadata()', () => {
+    it('should specify a plugin name of sayt-driver', () => {
+      expect(Driver.metadata.name).to.equal('sayt_driver');
+    });
+
+    it('should have two dependencies: "dom_events" and "sayt"', () => {
+      expect(Driver.metadata.depends).to.deep.equal(['dom_events', 'sayt']);
+    });
+  });
+
+  describe('.register()', () => {
+    beforeEach(() => {
+      Driver = {};
+    })
+
+    it('should keep a reference to all registered plugins', () => {
+      const Plugins = {
+        a: () => 'a',
+        b: () => true,
+      }
+      Driver = new SaytDriverPlugin();
+
+      expect(Driver.core).to.be.undefined;
+      Driver.register(Plugins);
+
+      expect(Driver.core).to.deep.equal(Plugins);
+    });
+  });
+
+  describe('.ready()', () => {
+    let registerListener: any;
+
+    beforeEach(() => {
+      registerListener = spy(Driver.core['dom_events'], 'registerListener');
+    });
+
+    it('should register an event listener for receiving sayt API requests', () => {
+      Driver.ready();
+
+      registerListener.getCall(0).calledWith(Driver.saytDataEvent, Driver.fetchSaytData);
+    });
+  });
+
+  describe('.unregister()', () => {
+    let unregisterListener;
+
+    beforeEach(() => {
+      unregisterListener = spy(Driver.core['dom_events'], 'unregisterListener');
+    });
+
+    it('should unregister the sayt event listener', () => {
+      Driver.unregister();
+
+      expect(unregisterListener).to.have.been.calledWith(Driver.saytDataEvent, Driver.fetchSaytData);
+    });
+  });
+
+  describe('.autocompleteCallback', () => {
```

Comment:
Keep it consistent. Add the `()`.

```suggestion
  describe('autocompleteCallback()', () => {
```